caddyhttp/fastcgi/fastcgi_test.go

```diff
@@ -51,6 +52,106 @@ func TestServeHTTP(t *testing.T) {
 	}
 }
 
+// listenerWithConnectionCounter adds a counter to keep track
+// of the number of accepted connections.
+type listenerWithConnectionCounter struct {
+	net.Listener
+	sync.RWMutex
+	ConnectionsAcceptedCounter int
```

Comment:
I prefer descriptive names. It could also be a counter for closed connections or whatever. But, ok, if you think a comment suits this purpose better than a descriptive name, I'll change it.